tests/epyccel/test_epyccel_lists.py

```diff
@@ -801,3 +802,13 @@ def list_return():
     assert python_result == pyccel_result
     assert isinstance(python_result, type(pyccel_result))
     assert isinstance(python_result.pop(), type(pyccel_result.pop()))
+
+def test_list_str(stc_language):
+    def list_str():
+        a = ['hello', 'world', '!']
+        return len(a)
```

Comment:
This test, does not test much beyond the compilation but better tests would require fixing one or more of:
- Returning strings (#2131)
- String comparison (#2196)
- Multi-level container shape (#2180)